packages/react-native-web/src/exports/Picker/index.js

```diff
@@ -14,65 +14,68 @@ import createElement from '../createElement';
 import useMergeRefs from '../../modules/useMergeRefs';
 import usePlatformMethods from '../../modules/usePlatformMethods';
 import PickerItem from './PickerItem';
-import StyleSheet, { type StyleObj } from '../StyleSheet';
+import StyleSheet from '../StyleSheet';
 import { forwardRef, useRef } from 'react';
 
 type PickerProps = {
   ...ViewProps,
-  children?: PickerItem | Array<typeof PickerItem>,
+  children?: typeof PickerItem | Array<typeof PickerItem>,
   enabled?: boolean,
   onValueChange?: (number | string, number) => void,
   selectedValue?: number | string,
-  style?: StyleObj,
+  style?: any,
   /* compat */
-  itemStyle?: StyleObj,
+  itemStyle?: any,
   mode?: string,
   prompt?: string
 };
 
-const Picker = forwardRef<PickerProps, *>((props, forwardedRef) => {
-  const {
-    children,
-    enabled,
-    onValueChange,
-    selectedValue,
-    style,
-    testID,
-    /* eslint-disable */
-    itemStyle,
-    mode,
-    prompt,
-    /* eslint-enable */
-    ...other
-  } = props;
+const Picker: React$AbstractComponent<PickerProps, mixed> = forwardRef<PickerProps, *>(
```

Comment:
Why `mixed` here? What's the value of not using `mixed` on all the other components?